guide/sections/part2/wis2node.adoc

```diff
@@ -8,6 +8,8 @@ Registration and decommissioning of WIS2 Nodes must be approved by the Permanent
 
 A WIS2 Node can be registered to exchange data concerning a WMO project or campaign, for a limited time. The National Focal Point can register such a project-related WIS2 Node in coordination with the Secretariat.
 
+A WIS2 Node can act as a regional publication facility on behalf of other centres.  Data or metadata publication by a regional publication facility will use the centre identifier of the data publisher.
```

Comment:
"regional publication facility" is not a concept that exists in WMO. The correct name is a DCPC (the DC part).
And "data publisher" might be confusing. I guess we are speaking about the "data producer" / "data originator". No ?

What about:

A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection role, as defined in the Manual on WIS. Data or metadata publication by a Data Collection will use the centre identifiers of the data producers.

Reply:
OK.  Updated to:

A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection role, as defined in the Manual on WIS. Data or metadata publication by a Data Collection role will use the centre identifiers of the data producers.